pkg/diagnostics/cluster/rolebindings.go

```diff
@@ -0,0 +1,99 @@
+package cluster
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	kerrs "k8s.io/kubernetes/pkg/api/errors"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	osclient "github.com/openshift/origin/pkg/client"
+	policycmd "github.com/openshift/origin/pkg/cmd/admin/policy"
+	"github.com/openshift/origin/pkg/diagnostics/types"
+)
+
+// ClusterRoleBindings is a Diagnostic to check that the default cluster role bindings match expectations
+type ClusterRoleBindings struct {
+	ClusterRoleBindingsClient osclient.ClusterRoleBindingsInterface
+	SARClient                 osclient.SubjectAccessReviews
+}
+
+const (
+	ClusterRoleBindingsName = "ClusterRoleBindings"
+)
+
+func (d *ClusterRoleBindings) Name() string {
+	return ClusterRoleBindingsName
+}
+
+func (d *ClusterRoleBindings) Description() string {
+	return "Check that the ClusterRoleBindings are up-to-date"
```

Comment:
Could this be slightly more descriptive for admins who are policy ignoramuses? Might be the first time they ever hear of it.

Maybe just "Check if the ClusterRoleBindings need updating for the current cluster version"